### PR TITLE
Added geop project with scala geoprocessing.

### PR DESCRIPTION
**Pull request by lossyrob**
This PR adds the scala project that will do geoprocessing using spark/geotrellis.

### CODE REVIEW

File: src/geop/sbt [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+  local target=$(readlink "$path")
+  if [[ "${target:0:1}" == "/" ]]; then
+    echo "$target"
+  else
+    echo "$(dirname $path)/$target"
+  fi
+}
+
+# a ham-fisted attempt to move some memory settings in concert
+# so they need not be dicked around with individually.
```
Comment (lliss) [inline src/geop/sbt]: I'd prefer to reword this comment since this is a public repo.

**Comment by jamesmcclain** [comment]
Is it okay to close this PR?

**Comment by mmcfarland** [comment]
Yeah, looks like we've moved beyond this by now.

**Comment by jamesmcclain** [comment]
Okay, I am going to go ahead and close it.

**Pull request closed without merging**